.github/workflows/ze-theia-slim.yaml

```
@@ -7,9 +7,9 @@ on:
       - .github/workflows/docker-reusable.yaml
   workflow_dispatch:
     inputs:
-      theia-version:
-        description: "Specify Theia version (e.g., 1.37)"
-        default: "latest"
+      docker-tags:
+        description: "Docker tags to build (e.g., 1.37 latest)"
```

Can docker tags be specified with space as the delimiter? If so, feel free to disregard this suggestion.
```suggestion
        description: "Docker tags to build (e.g., 1.37, latest)"
```
If multiple can be specified at once, an alternative would be to put it in a code block, but not required:
```suggestion
        description: "Docker tags to build (space-separated - e.g., `1.37 latest`)"
```